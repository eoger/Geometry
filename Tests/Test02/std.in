C
C Cercle
C Cercle 1
C Cercle 1 5
C Cercle 1 5 5 5
C C_ercle 1 5 10
C *Cercle 1 5 10
C Cercle ab1 5 10
C Cercle ab 5 10
C Cercle -ab 5 10
C Cercle 1 ab5 10
C Cercle 1 ab 10
C Cercle 1 -ab 10
C Cercle --1 5 10
C Cercle ++1 5 10
C Cercle 1 --5 10
C Cercle 1 ++5 10
C Cercle 1 5 -10
C Cercle 1 5 a
C Cercle 1 5 -a
C Cercle -9223372036854775809 5 5
C Cercle 9223372036854775808 5 5
C Cercle 5 -9223372036854775809 5
C Cercle 5 9223372036854775808 5
C Cercle 5 1 18446744073709551616
C Cercle 1 5 0
EXIT