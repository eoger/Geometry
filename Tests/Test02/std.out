ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
ERR
OK